### ISSUE TITLE
Link Falco rules to related packages

### ISSUE THREAD

**Issue by tegioz** [artifacthub/hub#123]

**Comment by tegioz** [comment]
Closed in favor of #186

**Comment by kris-nova** [comment]
Hi - just wondering what this means? and more importantly why the Falco community wasn't involved with any of this?

**Comment by tegioz** [comment]
Hi @kris-nova

This ticket was originally created to explore the idea of allowing users to discover Falco rules when browsing other related packages. Imagine a user is going through the details of a PostgresSQL package (Helm Chart, Operator, etc) and somehow we let the user know that there are some related Falco rules that might be relevant. Later on I closed it temporarily because that I thought that subject deserved a more in depth discussion. And recently, I saw a similar concept mentioned in falcosecurity/falco#1112, which led me to think it could actually be an interesting idea and I decided to reopen it.

About your second question, not sure what you really mean by "any of this", but I'll try to answer it.

@cynthia-sg and I have been working for the last couple of months on building a Hub that could potentially be the home of multiple kind of artifacts, which could be Helm Charts, Operators from OperatorHub, Falco rules, OPA policies, or any other. With that in mind, we've built the prototype you've seen at artifacthub.io.

So far we've worked only one day on Falco specific code and it was mostly to illustrate that the Hub wasn't built specifically for Helm Charts and that it was possible to add support for more package kinds easily. That Falco specific code is only a small script to load the rules available in the security hub, called intentionally security hub proof of concept, and some minor tweaks to the UI to display Falco/OPA specific content. It was meant as an experiment, nothing serious, so probably it wasn't worth involving or bothering anyone about it.

Our intention with this prototype was to have a foundation to keep building on with the feedback and collaboration from the  related projects communities. And now is probably a great time to start having those conversations and discuss how/if the ArtifactHub can be beneficial to other projects, what features they'd like to have, etc. Nothing is set on stone, this is just a prototype which hopefully can be easily modeled and extended with the help of the community to become something the users find valuable.

Hope this helps.
